### PR TITLE
Create links in Quizard homepage navbar to navigate to sections

### PR DESCRIPTION
**Pull request by whalewalker**
…specific ids

### CODE REVIEW

File: components/ui/organisms/Header.jsx [hunk 1]
```diff
@@ -7,23 +7,23 @@ export default function Header() {
     <header className="bg-[#C4FBDD] bg-[url(/assets/noise.svg)] p-5 lg:px-32 sticky top-0 z-50">
       <div className="max-w-7xl mx-auto flex justify-between items-center font-medium">
         {/* Logo */}
-        <div className="h-6 lg:h-max">
+        <a href="#" className="h-6 lg:h-max cursor-pointer">
```
Comment (CeoFred) [inline components/ui/organisms/Header.jsx:10]: should start with "/"

File: components/ui/organisms/Header.jsx [hunk 2]
```diff
@@ -7,23 +7,23 @@ export default function Header() {
...
         </div>
 
-        <nav className="hidden lg:flex font-medium space-x-5 items-center text-[#00210E]">
-          <div>Features</div>
-          <div>About Us</div>
-          <div>FAQ</div>
+        <nav className="hidden lg:flex font-medium space-x-5 items-center text-[#00210E] cursor-pointer">
+          <a href="#features">Features</a>
+          <a href="#about-us">About Us</a>
+          <a href="#faq">FAQ</a>
```
Comment (CeoFred) [inline components/ui/organisms/Header.jsx:26]: should start with "/"

**Review by CeoFred — CHANGES_REQUESTED** [review]

File: components/ui/organisms/Header.jsx (hunk 2 above)
Comment (CeoFred) [reply components/ui/organisms/Header.jsx:26]: @whalewalker the jOIN THE WAILIST button should also lead to /#

Comment (whalewalker) [reply components/ui/organisms/Header.jsx:26]: Alright, i will update the PR

File: components/ui/organisms/Header.jsx (hunk 1 above)
Comment (whalewalker) [reply components/ui/organisms/Header.jsx:10]: OK

**Comment by dpkreativ** [comment]
[Create links in Quizard homepage navbar to navigate to sections](https://trello.com/c/MoyXz2oA/18-create-links-in-quizard-homepage-navbar-to-navigate-to-sections)

File: components/ui/organisms/Header.jsx (hunk 2 above)
Comment (whalewalker) [reply components/ui/organisms/Header.jsx:26]: Alright, i will look into this

**Review by dpkreativ — APPROVED** [review]
Good work @whalewalker 🤗

**Review by dpkreativ — APPROVED** [review]

**Comment by dpkreativ** [comment]
Hey @whalewalker, please assign this PR's review to me so I can merge your PR. Thanks

**Review by CeoFred — APPROVED** [review]

**Pull request merged**